### PR TITLE
Promote node-driver-registrar and livenessprobe staging images to release

### PR DESCRIPTION
**Pull request by mauriciopoppe**
Generated with step 14 of https://github.com/kubernetes-csi/csi-release-tools/blob/master/SIDECAR_RELEASE_PROCESS.md#release-process

```
./generate.sh > images.yaml
```

/cc @msau42 
/assign @msau42

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml [hunk 1]
```diff
@@ -83,6 +84,7 @@
     "sha256:19063babf32ef0ddde9253e703be469d5ca390700e8302095bad9e9de7f2438b": [ "v4.2.0" ]
     "sha256:818f35653f2e214db81d655063e81995de9073328a3430498624c140881026a3": [ "v4.2.1" ]
     "sha256:ac845016480ce5eb18a04c1e4e7a4613900e662367dd1771db1e4ccfcedda5e2": [ "v5.0.0" ]
+    "sha256:001943f2475ba4015af2b4fa3df6a5a95ea581c4ba5932f9f244e7def71f4a4b": [ "v5.0.0-test-main" ]
```
Comment (msau42) [inline k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:87]: I don't think we want to promote this. Can we modify the script to also check for the end of the tag?

Comment (mauriciopoppe) [reply k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:87]: sure, looks like this worked:

```
gcloud container images list-tags gcr.io/k8s-staging-sig-storage/csi-snapshotter --format="get(digest,tags)" --sort-by="tags" --filter="tags~^v\d+\.\d+\.\d+\$"
sha256:7f93019f399454598370aab2cc1a652a3b8ff722ab92ec4ca9db0eed53acc4b4 v1.1.0
sha256:4e78bb0116555a10ef59c1cdf0648b06b483903a1919def5e3c7ba68b9fa08cb v1.2.0
sha256:46ec7b441027cb6db9f7a5b3355623c18fe7da9d4095c2816f48ffc8ad47c170 v1.2.2
sha256:1530db9fd380ecee86b48a7b3540568c46adb5a64b7fe718c51d9260c4834fd8 v2.0.1
sha256:35ead85dd09aa8cc612fdb598d4e0e2f048bef816f1b74df5eeab67cd21b10aa v2.1.0
sha256:de94e695fc27ee421da8b9932c3d42699fe7b848bc34717f67f4350946bc46c9 v2.1.1
sha256:f0673197110ba3c38a94b42fc0037d0c209b9d17c0b796072d26df72195a7ecd v2.1.2
sha256:ad9ebd1db27bc64e0d9ccac3d945fa598b7e5213aa6def217a10d6a4ae6a9770 v2.1.3
sha256:2d719cc8f5f1d3794ee6ae9bc2f93820e78ff34908ab05ed94293e5b7ddebfb0 v2.1.4
sha256:74e5a1c97b0c558fb0c59ea002b6ddc98a236676221030306355ed1423df667e v2.1.5
sha256:ece29e2217e0faf87973565c062c8508c2b78039169e855032bb357acefa3c98 v3.0.0
sha256:e5ce22ddc2a6fbc9b5729dbcfaf1159c4b0d48dad62ddd1b93503c1348e4a406 v3.0.1
sha256:26e327b018c21a49523b759d7787e99553181ae9ef90b6bdc13abe362a43ced0 v3.0.2
sha256:9af9bf28430b00a0cedeb2ec29acadce45e6afcecd8bdf31c793c624cfa75fa7 v3.0.3
sha256:51f2dfde5bccac7854b3704689506aeecfb793328427b91115ba253a93e60782 v4.0.0
sha256:e27ad21f007bae28eef68934a4834ca0a5c669e4c246c4900378e7ed1680c37e v4.1.0
sha256:ed98431376c9e944e19a465fe8ea944806714dd95416a0821096c78d66b579bd v4.1.1
sha256:19063babf32ef0ddde9253e703be469d5ca390700e8302095bad9e9de7f2438b v4.2.0
sha256:818f35653f2e214db81d655063e81995de9073328a3430498624c140881026a3 v4.2.1
sha256:ac845016480ce5eb18a04c1e4e7a4613900e662367dd1771db1e4ccfcedda5e2 v5.0.0
sha256:89e900a160a986a1a7a4eba7f5259e510398fa87ca9b8a729e7dec59e04c7709 v5.0.1
```

File: k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml [hunk 2]
```diff
@@ -99,7 +100,7 @@
     "sha256:6029c252dae6178c99b580de72d7776158edbc81be0de15cedc4152a3acfed18": [ "v1.7.3" ]
 - name: iscsiplugin
   dmap:
-    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "v0.1.0" ]
+    "sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de": [ "v0.1.0" ]
```
Comment (mauriciopoppe) [inline k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:103]: this is a little bit concerning, the image must have been created again

Comment (mauriciopoppe) [reply k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:103]: error log from `pull-k8sio-cip`

```
level=fatal msg="run `cip run`: promoting images: [edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {asia.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {eu.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {us.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping]"
```

Comment (mauriciopoppe) [reply k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:103]: Opened https://github.com/kubernetes-csi/csi-driver-iscsi/issues/97 to track this

Comment (mauriciopoppe) [reply k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml]: done

**Pull request merged**